### PR TITLE
upgraded node to the LTS version

### PR DESCRIPTION
**Pull request by allandegnan**
4.2.2 is the latests LTS version.  We could also use version 5 but LTS is probably a better bet for us.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
 FROM quay.io/ukhomeofficedigital/docker-centos-base
 
+ENV NODE_VERSION v4.2.2
```
Comment (vaijab) [inline Dockerfile]: Maybe move this line right before you actually need the variable? That'd save us from re-creating layers for things like `WORKDIR /opt/nodejs` which don't change.

Comment (allandegnan) [inline Dockerfile]: I changed it but honestly I'd rather the file were more readble than efficant when it comes to versions like that.

**Comment by PurpleBooth** [comment]
Looks fine but please bring branch up to date.

**Comment by jon-shanks** [comment]
LGTM though probably want an ONBUILD trigger in there to at least patch on downstream images using yours when they go to use it i.e. OS patch

File: Dockerfile [hunk 2]
```diff
@@ -3,6 +3,8 @@ FROM quay.io/ukhomeofficedigital/docker-centos-base
 RUN mkdir -p /opt/nodejs /app
 
 WORKDIR /opt/nodejs
+
+ENV NODE_VERSION v4.2.2
 RUN yum install -y git curl && \
     curl https://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-x64.tar.gz | tar xz --strip-components=1
```
Comment (vaijab) [inline Dockerfile]: So the `NODE_VERSION` is not being used anymore?

**Comment by PurpleBooth** [comment]
LGTM

**Pull request merged**